docs/pages/overview.md

```diff
@@ -20,13 +20,15 @@ Meshery provides this high-level functionality:
 
 
 <h2>What challenges does Meshery solve?</h2>
-<p style="text-align:center;"><b>Service mesh management - one or multiple service meshes.</b></p>
+<b>Service mesh management - one or multiple service meshes.</b>
 
-<p style="margin-bottom:1em; margin-top:1em;">Anytime performance questions are to be answered, they are subjective to the specific workload and infrastructure used for measurement. Given this challenge, the Envoy project, for example, refuses to publish performance data because such tests can be 1) involved and 2) misinterpreted.</p>
+Anytime performance questions are to be answered, they are subjective to the specific workload and infrastructure used for measurement. Given this challenge, the Envoy project, for example, refuses to publish performance data because such tests can be:
+- Involved
+- Misinterpreted
 
-<p style="margin-bottom:1em; margin-top:1em;">Beyond the need for performance and overhead data under a permutation of different workloads (applications) and types and sizes of infrastructure resources, the need for cross-project, apple-to-apple comparisons are also desired in order to facilitate a comparison of behavioral differences between service meshes and selection of their use. Individual projects shy from publishing test results of other, competing service meshes. An independent, unbiased, credible analysis is needed.<br /></p>
+Beyond the need for performance and overhead data under a permutation of different workloads (applications) and types and sizes of infrastructure resources, the need for cross-project, apple-to-apple comparisons are also desired in order to facilitate a comparison of behavioral differences between service meshes and selection of their use. Individual projects shy from publishing test results of other, competing service meshes. An independent, unbiased, credible analysis is needed.
 
-<p style="margin-bottom:1em; margin-top:1em;">Meshery is intended to be a vendor and project-neutral utility for uniformly benchmarking the performance of service meshes. Between service mesh and proxy projects (and surprisingly, within a single project), a number of different tools and results exist. Meshery allows you to pick an efficient set of tools for your ecosystem by providing performance evaluation and metrics.<br /></p>
```

Comment:
the additional white space was included to help the readers eyes flow from one section to the next. We can change, if you think this reads better.

Reply:
The spacing was not working, the default spacing provided by kramdown engine is basically the same to the margins specified. Hence removing them will have no change in look.

Reply:
Roger that. Sounds good.